ReShade/Shaders/Otis/MouseOverlay.fx

```diff
@@ -0,0 +1,33 @@
+///////////////////////////////////////////////////////////////////
+// This effects simply shows the mouse coordinates on the screen. It can be used in combination
+// of effects which use the mousecoordinates supplied by ReShade so you can better see what
+// you're doing.
+///////////////////////////////////////////////////////////////////
+
+#include EFFECT_CONFIG(Otis)
+#include "Common.fx"
+
+#if USE_MOUSEOVERLAY
+
+#pragma message "MouseOverlay by Otis\n"
+
+namespace Otis
+{
+	float4 PS_MouseCoordOverlay(float4 vpos:SV_Position, float2 texcoord: TEXCOORD) :SV_Target
+	{
+		float2 focusPoint = ReShade::MouseCoords * ReShade::PixelSize;
+		if(abs(focusPoint.x - texcoord.x) < MOL_CursorSize && abs(focusPoint.y - texcoord.y) < MOL_CursorSize)
```

Comment:
That at least doesn't give a compile errors, and makes sense indeed. But the problem is: this also makes the shader do nothing (always return backbuffer pixel, so expression is always false). 

I've confirmed vpos does indeed contains the screencoords:
```c
return tex2D(ReShade::BackBuffer, vpos.xy * ReShade::PixelSize);
```
returns the same framebuffer. If I replace that with a color, it is indeed called, so that's not it.

If I remove the call to abs, I do see a nice red square in the right/bottom quadrant. What I wanted with the abs is to avoid some comparisons, but with a float2 this apparently goes wrong. Could this be a result of floating point comparisons against 0 ? If a `float operator value` predicate is issued inside the IIF compound statement it won't be evaluated to a float before comparison, but with the all/any on the abs it will, and all/any check on != 0, which in this case could be a problem as comparing to 0 using floats has problems. 

Just guessing here, I am not a shader expert/hlsl expert...